examples/db-templates/postgresql-persistent-template.json

```diff
@@ -158,13 +166,21 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
+      "description": "Maximum amount of memory a pod can use; if it exceeds the specified memory limit, offending containers will be terminated and potentially restarted dependent upon the container restart policy",
```

Comment:
I don't think this description is accurate.  it won't be terminated, it simply won't be able to get access to more memory than this (which for some applications might cause them to terminate themselves, but it's not openshift doing it)